src/bun.js/webcore/prompt.zig

```diff
@@ -260,6 +389,177 @@ pub const prompt = struct {
         };
 
         // 7. Pause while waiting for the user's response.
+        if (comptime !Environment.isWindows) {
+            const c_termios = @cImport({
+                @cInclude("termios.h");
+                @cInclude("unistd.h");
+                @cInclude("signal.h");
+            });
+
+            if (c_termios.isatty(bun.FD.stdin().native()) == 1) {
```

Comment:
```suggestion
            if (bun.c.isatty(bun.FD.stdin().native()) == 1) {
```

also let's invert this check and break out of this block if it's false. this means all the code inside of the `if` will have one less indent